packages/docs/src/components/prism.js

```diff
@@ -0,0 +1,21 @@
+/*
+  This theme object is based on the prism-react-renderer
+  shape. It doesn't pull theme values from the config file
+  so it needs to use hardcoded values or the object directly.
+
+  SOURCE: https://github.com/FormidableLabs/prism-react-renderer/blob/master/themes/duotoneDark.js
+*/
+export default {
+  plain: {
+    color: 'black',
+  },
+  styles: [
+    {
+      types: ['comment', 'prolog', 'doctype', 'cdata'],
+      style: {
+        color: '#777',
+        fontStyle: 'italic',
+      },
```

Comment:
Since this is based on the *props* in the `prism-react-renderer` component, I wonder if we should stick to using styles provided by Emotion – I might be misreading where this is imported and used